nucleus/common/common-util/src/main/java/com/sun/enterprise/util/io/FileUtils.java

```diff
@@ -1032,6 +1033,37 @@ public static File[] findFilesInDir(File dir, final String regexp) {
         return matches;
     }
 
+
+    /**
+     * Convert a URL to an absolute path if it is possible..
+     *
+     * @param resourceUrl
+     * @return absolute path to the file or directory pointed by the URL
+     * @throws IllegalArgumentException if the URL cannot be converted to a File
+     *
+     */
+    public static String toAbsolutePath(URL resourceUrl) throws IllegalArgumentException {
```

Comment:
This method is not used. And that's correct - I would always prefer to explicitly call `toFile(URL).getPath()` for clarify.
Can we delete this method?

Reply:
It is used in 7.1 branch, this is backport. But ok, I will rebase that later and remove it there too.